src/main/java/com/listywave/user/repository/custom/impl/UserRepositoryImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.listywave.user.repository.custom.impl;
+
+import static com.listywave.list.application.domain.QItem.*;
+import static com.listywave.list.application.domain.QLists.*;
+
+import com.listywave.list.application.domain.CategoryType;
+import com.listywave.list.application.domain.Lists;
+import com.listywave.user.repository.custom.UserRepositoryCustom;
+import com.querydsl.core.types.Predicate;
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class UserRepositoryImpl implements UserRepositoryCustom {
+    private final JPAQueryFactory queryFactory;
+    @Override
+    public List<Lists> findFeedLists(Long userId, String type, CategoryType category, Long cursorId, int size) {
+        List<Lists> fetch = queryFactory
+                .select(lists)
+                .from(lists)
+                .leftJoin(lists.items, item)
+                .where(
+                        userIdEq(userId),
+                        typeEq(type),
+                        categoryEq(category),
+                        listIdGt(cursorId)
+                )
+                .distinct()
+                .limit(size + 1)
+                .orderBy(lists.updatedDate.desc())
+                .fetch();
+        return fetch;
+    }
+
+    private BooleanExpression listIdGt(Long cursorId) {
+        return cursorId == null ? null : lists.id.lt(cursorId);
+    }
+
+    private BooleanExpression categoryEq(CategoryType categoryCode) {
+
+        return "0".equals(categoryCode.getCodeValue()) ? null : lists.category.eq(categoryCode);
```

Comment:
개인적으로 삼항연산자보다 아래처럼 if문 쓰는게 가독성이 더 좋은데 어떻게 생각하시나요?
```suggestion
if ("0".equals(categoryCode.getCodeValue())) {
    return null;
}
return lists.category.eq(categoryCode);
```

Reply:
해당 제안 주신 부분이 가독성 면에서 제안 주신게 더 좋은 거 같습니다!